indico/modules/events/abstracts/forms.py

```diff
@@ -413,8 +413,8 @@ class CreateEmailTemplateForm(EditEmailTemplateRuleForm):
 
 class AbstractForm(IndicoForm):
     title = StringField(_("Title"), [DataRequired()])
-    description = IndicoMarkdownField(_('Content'), editor=True, mathjax=True)
-    submitted_contrib_type = QuerySelectField(_("Type"), get_label='name', allow_blank=True,
+    description = IndicoMarkdownField(_('Content'), [DataRequired()], editor=True, mathjax=True)
+    submitted_contrib_type = QuerySelectField(_("Contribution type"), get_label='name', allow_blank=True,
```

Comment:
I think we made it clear when I opened the pull request that we want to rename it to "Contribution type" because in legacy it was "Presentation type". It also makes the transition from an abstract to a contribution easier for the user.